### ISSUE TITLE
CMIP6_CV.json : most piClim-* experiments miss some allowed components

### ISSUE THREAD

**Issue by senesis** [WCRP-CMIP/CMIP6_CVs#592]
piClim-control has allowed components : AER, BGC, CHEM
As most piClim-* experiments are sensitivity experiments w.r.t. this one, they need to have the same allowed components list.  This is not the case, see e.g. piClim-ghg, piCLim-2xDMS ....

**Comment by durack1** [comment]
@senesis, reviewing these in [CMIP6_experiment_id.html](https://rawgit.com/WCRP-CMIP/CMIP6_CVs/master/src/CMIP6_experiment_id.html) you'll note that many `activity_id`'s or MIPs are requesting these 31 `piClim*` experiments, so what specific changes would you want to see implemented? For e.g. `piClim-anthro` is requested by `RFMIP` so would make no sense for `BGC` to be added for example

**Comment by senesis** [comment]
We, at CNRM, will do one single piClim-control, which has all three allowed components (AER, BGC and CHEM) activated; 

So, our piClim-anthro should activate the same components, in order to get a sensible comparison with the control; in that sense, it do make sense to add BGC to piClim-anthro. 

And you cannot expect that labs will do multiple piClim-control, one for each sensitivity experiment components setting; 

So you should add in each piClim experiment, as allowed components, all components allowed in piClim-control

**Comment by taylor13** [comment]
quite right that for coupled runs, the same components need to be active in the perturbed and control runs.  

For the piClim expts. where "BGC" is missing, these are AMIP-style experiments.  I think the "BGC" component could be active in these (in the sense that you might keep track of fluxes of carbon into and out of land and atmosphere).  I don't think, however, you want vegetation to evolve in these runs.  Users will have to run their models in a sensible way, and they should be allowed to include BGC if that makes sense.

Bottom line is: we need to add "BGC" to allowed components for piClim-ghg, piClim-histaer, piClim-histall, piClim-histghg, piClim-histnat, and piClim-lu .  

@senesis do we need to do anything  else?

**Comment by senesis** [comment]
No, the bottom line is fine. Thanks

**Comment by durack1** [comment]
@senesis thanks for this suggestion, these updates are now in place. Take a peek at [CMIP6_experiment_id.html](https://rawgit.com/WCRP-CMIP/CMIP6_CVs/master/src/CMIP6_experiment_id.html) and let me know of any further issues
